tortoise/validators.py

```diff
@@ -66,12 +66,12 @@ class MinValueValidator(Validator):
     """
 
     def __init__(self, min_value: Union[int, float]):
-        if min_value is None or type(min_value) not in [int, float]:
+        if type(min_value) not in [int, float]:
```

Comment:
What about replace `type` to `isinstance`?

Reply:
i was about to ask you, I'm going to do it